### ISSUE TITLE
RCS1179 False Negative

### ISSUE THREAD

**Issue by nnpcYvIVl** [JosefPihrt/Roslynator#449]
Should RCS1179 offer to change this:

```csharp
namespace RCS1179_False_Negative
{
    static class Program
    {
        public static string Method()
        {
            string value = "1";
            if (System.Environment.Is64BitProcess)
            {
                value = "2";
            }

            return value;
        }
    }
}
```

To this?

```csharp
namespace RCS1179_False_Negative
{
    static class Program
    {
        public static string Method()
        {
            if (System.Environment.Is64BitProcess)
            {
                return "2";
            }

            return "1";
        }
    }
}
```

**Comment by JosefPihrt** [comment]
It was originally reported but it was removed here 360e56d9cafe83936a1985b73ad3afa3ded6cae0

**Comment by nnpcYvIVl** [comment]
Got it, thanks.
